SpeechResponder/Properties/SpeechResponder.resx

```diff
@@ -230,4 +230,10 @@
   <data name="reset_script_message" xml:space="preserve">
     <value>Are you sure you want to reset the "{0}" script?</value>
   </data>
+  <data name="messagebox_recoveredScript_title" xml:space="preserve">
+    <value>Recovered Script</value>
+  </data>
+  <data name="messagebox_recoveredScript" xml:space="preserve">
+    <value>EDDI has recovered a script after the application has Crashed. Do you want to open it?</value>
```

Comment:
Please lowercase `crashed`. ;-)

Reply:
Or perhaps better, revise to something like `EDDI has recovered a script that was being edited when the program closed unexpectedly. Do you want to re-open it?`